### PR TITLE
Add Dev2 posting

### PR DESCRIPTION
**Pull request by IanWhitney**
Work in progress. Looking for feedback. Also the job posting isn't up yet & we can't merge this until it is.

This is mostly unchanged from our last posting

https://github.com/umn-asr/umn-asr.github.io/pull/17

I've added one new section, `What You'll Do` which outlines what things
the person will be doing in their first week, month, quarter, year. This
idea is 'borrowed' from

https://weworkremotely.com/remote-jobs/respondent-io-people-operations

The goal of this section is two-fold. First we want to give a good
picture of what the job entails, as that's a question we've gotten
before. Second, I wanted to show the that we intend to train and grow
the person we hire.

### CODE REVIEW

**Comment by IanWhitney** [comment]
@umn-asr/asr-custom-developers

**Comment by IanWhitney** [comment]
Ran this through http://joblint.org and it flagged no problems.

But http://gender-decoder.katmatfield.com/results/372784a6-4297-4e15-930a-cf743a349c43 shows some biased word usage, so I'll rework that.

**Comment by IanWhitney** [comment]
Ah, the gender-decoder site was flagging my use of the word "driven", but it couldn't notice that I was using it in the phrase "Test-driven development". So, that looks ok.

File: jobs.markdown [hunk 1]
```diff
@@ -4,4 +4,70 @@ title: Jobs
...
+
+_In your first month_ you'll be pairing with other members of the team, using test-driven development to deliver features. You'll help resolve problem tickets along with an experienced team member. And you'll attend some of the University's [coding meetups](http://code-people.umn.edu/) and [professional groups](https://campus-climate.umn.edu/get-involved#faculty-staff)
+
+_In your first quarter_ you'll learn how to write robust, fast tests in RSpec and Jest while pairing on both new and long-lived Rails and JavaScript applications. You'll explore new technologies and meet the UMN IT community at events like [Hacker Hours](https://umnhackerhours.github.io), [Tech-People](http://tech-people.umn.edu), and [hackathons](http://umn.campuscodefest.org/).
+
+_In your first year_ you'll become an experienced pair programmer familiar with the full process of application development. You'll use [Ansible](https://www.ansible.com) and [Docker](https://www.docker.com) to manage infrastructure. You'll learn about Continuous Integration and Delivery to test and deploy the features you and your pair wrote.
```
Comment (davinlagerroos) [inline jobs.markdown]: "You'll learn about CI/CD to test and deploy ..." reads a little awkward. Maybe "You'll learn about using ....", or just "You'll use ..."?

File: jobs.markdown [hunk 2]
```diff
@@ -4,4 +4,70 @@ title: Jobs
...
+- A cover letter
+  - Explain why you're interested in the job and why you're a good candidate
+- Names and contact information for three references
+  - Mentors, bosses, co-workers, people you study with or talk to at meetups
+
+**We place a lot of value on the cover letter**. Let us say that again: **_we place a lot of value on the cover letter_**. Tell us what excites you about this job, work you've done and where you want your career to go. This is more interesting to us than a list of technologies you've used.
+
+You can take a break at any point during the application process. Click the **Save Draft** button and you can return to your application later. When you log back into the job application system, click **My Activities** and then click on the job title **Developer 2 (9790D2) - Temporary/2 years**
+
+If you have any questions while applying, look at the <a href="http://humanresources.umn.edu/find-job/using-employment-system">Using the Job Application System</a> page.
```
Comment (davinlagerroos) [inline jobs.markdown]: Is there a reason why we are using an HTML anchor tag instead of the markdown syntax here?

File: jobs.markdown [hunk 3]
```diff
@@ -4,4 +4,70 @@ title: Jobs
...
+
+We use three interviews. 
+
+1. Meet with a small group of ASR employees to discuss the job, your previous experience, and your qualifications.
+1. A short take-home exercise
+1. Pair programming with Custom Solutions developers using your programming language of choice.
+
+Oh, and we won't ask you to code anything on a whiteboard.
+
+Each interview takes about 2 hours and will be on different days. We can also arrange for remote interviews.
```
Comment (davinlagerroos) [inline jobs.markdown]: Do we want to say anything about the possibility of combining interviews? (do we want to try to combine interviews?)

File: jobs.markdown [hunk 4]
```diff
@@ -4,4 +4,70 @@ title: Jobs
...
+
+_In your first month_ you'll be pairing with other members of the team, using test-driven development to deliver features. You'll help resolve problem tickets along with an experienced team member. And you'll attend some of the University's [coding meetups](http://code-people.umn.edu/) and [professional groups](https://campus-climate.umn.edu/get-involved#faculty-staff)
+
+_In your first quarter_ you'll learn how to write robust, fast tests in RSpec and Jest while pairing on both new and long-lived Rails and JavaScript applications. You'll explore new technologies and meet the UMN IT community at events like [Hacker Hours](https://umnhackerhours.github.io), [Tech-People](http://tech-people.umn.edu), and [hackathons](http://umn.campuscodefest.org/).
+
+_In your first year_ you'll become an experienced pair programmer familiar with the full process of application development. You'll use [Ansible](https://www.ansible.com) and [Docker](https://www.docker.com) to manage infrastructure. You'll configure Continuous Integration and Delivery tools to test and deploy the features you and your pair wrote.
+
```
Comment (isen0011) [inline jobs.markdown]: I really like this "What you will do" section.

**Pull request merged**